### PR TITLE
Added report_url_for_resource to create URL for front-end mapping

### PR DESCRIPTION
**Pull request by rsdesoto**
Added `report_url_for_resource` - as part of work on https://app.clubhouse.io/imagineer/story/5566/solovis-entity-mapping-front-end

### CODE REVIEW

File: lib/solovis_api_client/site.rb [hunk 1]
```diff
@@ -114,6 +114,28 @@ def get_entities
     model_environment["entities"]
   end
 
+  def report_url_for_resource(resource_id)
+    if get_entities.find{|entity| entity["resourceId"] == resource_id}
```
Comment (antifun) [inline lib/solovis_api_client/site.rb]:
```suggestion
    if (entity = get_entities.find { |entity| entity["resourceId"] == resource_id })
      entity_id = entity["id"]
```

File: lib/solovis_api_client/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module SolovisApiClient
-  VERSION = "0.1.16"
+  VERSION = "0.1.17"
```
Comment (antifun) [inline lib/solovis_api_client/version.rb]: This doesn't belong in the PR. Should be done as a separate commit directly to master. If we're being pedantic. (This is a "release" thing, not a "merge PR" thing.)

Comment (rsdesoto) [reply lib/solovis_api_client/version.rb]: 👍

**Comment by rsdesoto** [comment]
@antifun - edits made
